dstep/translator/IncludeHandler.d

```diff
@@ -187,18 +188,6 @@ private:
             output.separator();
     }
 
-    void importsBlock(Output output, string[] imports)
-    {
-        import std.array : empty;
-        import std.algorithm.sorting : sort;
-
-        foreach (entry; imports.sort.filter!(e => !e.empty))
-            output.singleLine(entry);
-
-        if (!output.empty)
-            output.separator();
-    }
-
```

Comment:
I really don't like that you have one commit in a PR which added code and then another commit which restores that modifies/restores that code.